NFluidsynth/Native/LibFluidsynth.cs

```diff
@@ -19,7 +19,7 @@ static LibFluidsynth()
                 {
                     if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
                     {
-                        return NativeLibrary.Load("/home/pj/Projects/fluidsynth/build/src/libfluidsynth.so");
+                        return NativeLibrary.Load("libfluidsynth.so");
```

Comment:
Ah fuck.

Reply:
Yeah, I had to change it or it'd crash oof